framework/build.gradle

```diff
@@ -121,10 +123,12 @@ test {
     testLogging {
         exceptionFormat = 'full'
     }
+    /*
```

Comment:
Why annotate this？

Reply:
Use the report part configuration of the offlinsCoverage module

Reply:
Use the report part configuration of the offlinsCoverage module